Makefile

```diff
@@ -17,7 +17,8 @@ build: clean Makefile src/cs50.c src/cs50.h
 	gcc -o build/usr/local/lib/libcs50.a -shared build/cs50.o
 	rm -f build/cs50.o
 	cp src/cs50.h build/usr/local/include
-	chmod -R a+rX build
+	chmod -R 0755 build
```

Comment:
per `man find`:

> -exec command {} +
>               This  variant  of  the -exec action runs the specified command on the selected files, but the command line is built by appending each
>               selected file name at the end; the total number of invocations of the command will be much less than the  number  of  matched  files.
>               The  command line is built in much the same way that xargs builds its command lines.  Only one instance of `{}' is allowed within the
>               command.  The command is executed in the starting directory.

no problem. will change to `find build -type d -exec chmod 0755 {} +`.